src/lib/db/api-token-store.ts

```diff
@@ -36,45 +36,33 @@ interface ITokenRow extends ITokenInsert {
     project: string;
 }
 
-const createTokenRowReducer =
-    (allowOrphanedWildcardTokens: boolean) => (acc, tokenRow) => {
-        const { project, ...token } = tokenRow;
-        if (!acc[tokenRow.secret]) {
-            if (
```

Comment:
this is more than a flag removal

Reply:
In `!allowOrphanedWildcardTokens &&` removing this flag makes entire logic statement false (this kill-switch defaults to `true`)